adds TR in all missing places

DanielSvoboda opened the pull request:
add TR for [PR 700](https://github.com/shadps4-emu/shadPS4/pull/700)
tr("Can't apply cheats before the game is started")
![image](https://github.com/user-attachments/assets/1b6cac0a-bffe-448f-8673-a61ac0650675)

add: Icon,Name,Serial,Region,Firmware,Size,Version,Path
![image](https://github.com/user-attachments/assets/5073ac0c-e9e0-45d9-ab1f-ab77ea7904a4)

add: Restore Defaults, Save, Close, Apply
![image](https://github.com/user-attachments/assets/059a4f3b-0c76-424a-a391-a084cff11611)

also added more information when downloading patches:
"If the patch does not appear, it may be that it does not exist for the specific serial and version of the game. It may be necessary to update the game."
![image](https://github.com/user-attachments/assets/a4dd5884-8ab7-40fe-acf4-339206f61010)

*note, in fa_IR.ts I fixed the spacing, so it looks like I changed everything, but it was just the spaces and the separation of MainWindow.h and MainWindow.cpp

DanielSvoboda committed:
add TR for PR 700

DanielSvoboda committed:
Update src/qt_gui/translations/pl_PL.ts

Co-authored-by: Marcin Mitura <mitura07@wp.pl>

mavethee commented on src/qt_gui/translations/pl_PL.ts:
```diff
@@ -898,5 +898,10 @@
+		<message>
+			<location filename="../cheats_patches.cpp" line="1163"/>
+			<source>Can't apply cheats before the game is started</source>
+			<translation>Nie można zastosować cheatów przed rozpoczęciem gry.</translation>
```
```suggestion
			<translation>Nie można zastosować kodów przed uruchomieniem gry.</translation>
```

Keep translation consistent to other translated strings

DanielSvoboda committed:
add more information download patches

If the patch does not appear, it may be that it does not exist for the specific serial and version of the game. It may be necessary to update the game.

DanielSvoboda committed:
Merge branch 'main' of https://github.com/DanielSvoboda/shadPS4

DanielSvoboda committed:
Save,Apply,Restore Defaults,Close

Save,Apply,Restore Defaults,Close

DanielSvoboda committed:
fix PL text

DanielSvoboda committed:
Icon,Name,Serial,Region,Firmware,Size,Version,Path

DanielSvoboda committed:
Game List
